addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2641,7 +2641,7 @@ export class Wysiwyg extends Component {
                         .removeClass(id)
                         .popover({
                             trigger: 'hover',
-                            content: response.message.data.message || '',
+                            content: response.message.data?.message || '',
```

Comment:
I'm still not convinced about the changes :/
The problem occurs only when doing `Remove the CTA in 16.0 with the bin icon (on the button itself, or from the edit panel, next to the "Block" section)` because the `website.header_call_to_action` is still active but it's empty that's it ? (this info should be in the commit message).

If that's the case, the good fix should be done in the upgrade script so that it deactivates the CTA if it's empty no ?

Reply:
> the good fix should be done in the upgrade script so that it deactivates the CTA if it's empty no ?

No, because deleting the CTA through the trash icon actually deletes part of the arch (that's why this has been disabled in 17.0, because with the new options such behavior triggers template errors). It needs to be reset when updating if the header is to work, and it's not possible to reset it with something else than its default state (i.e. active).